scripts/configure

```diff
@@ -198,6 +198,11 @@ if [[ -f "${STATUS_DIR}/node-status-bitcoind-ready" ]]; then
   sed -i "s/bitcoin.node=.*/bitcoin.node=bitcoind/g;" "$LND_CONF_FILE"
 fi
 
+# If there was an UMBREL_HTTP_PORT set before then add it to the $ENV_FILE
+if [ ! -z $UMBREL_HTTP_PORT ]; then
```

Comment:
This will throw an unbound variable error if `UMBREL_HTTP_PORT` is not set.

Reply:
ok ill remove it